### ISSUE TITLE
Ranger crashed.

### ISSUE THREAD

**Issue by granhalcon** [ranger/ranger#2183]
#### Traceback

```
ranger version: ranger 1.9.3
Python version: 3.9.0 (default, Oct  7 2020, 23:09:01) [GCC 10.2.0]
Locale: es_MX.UTF-8
Current file: test.css

Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/usr/lib/python3.9/site-packages/ranger/core/fm.py", line 383, in loop
    ui.handle_input()
  File "/usr/lib/python3.9/site-packages/ranger/gui/ui.py", line 266, in handle_input
    self.handle_key(key)
  File "/usr/lib/python3.9/site-packages/ranger/gui/ui.py", line 196, in handle_key
    elif not DisplayableContainer.press(self, key):
  File "/usr/lib/python3.9/site-packages/ranger/gui/displayable.py", line 275, in press
    focused_obj.press(key)
  File "/usr/lib/python3.9/site-packages/ranger/gui/widgets/console.py", line 188, in press
    self.type_key(key)
File "/usr/lib/python3.9/site-packages/ranger/gui/widgets/console.py", line 214, in type_key
    self._answer_question(answer)
  File "/usr/lib/python3.9/site-packages/ranger/gui/widgets/console.py", line 197, in _answer_question
    callback(answer)
  File "/usr/lib/python3.9/site-packages/ranger/config/commands.py", line 759, in _question_callback
    self.fm.execute_file(files, label='trash')
  File "/usr/lib/python3.9/site-packages/ranger/core/actions.py", line 459, in execute_file
    filenames = [f.path for f in files]
  File "/usr/lib/python3.9/site-packages/ranger/core/actions.py", line 459, in <listcomp>
    filenames = [f.path for f in files]
AttributeError: 'str' object has no attribute 'path'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues

```

**Comment by toonn** [comment]
Hmm, could you check whether this is still an issue on master?

**Comment by kaykay38** [comment]
This still occurs on the master branch. Here are the details [Issue 2199](https://github.com/ranger/ranger/issues/2199).

**Comment by toonn** [comment]
Should be fixed by #1871.
